apps/platform/lib/jobs/deleteBranchJob.ts

```diff
@@ -0,0 +1,26 @@
+import { Job, PrismaTransaction } from "./baseJob";
+
+export class DeleteBranchJob extends Job {
+  constructor() {
+    super();
+  }
+
+  async handle(tx: PrismaTransaction) {
+    // Get all branches that have deletedAt within the past 24 hours.
+    const branches = await tx.branch.findMany({
```

Comment:
Here, we need to remove all other associated data as well like `secrets` and `pull requests`.